mappings/net/minecraft/datafixer/fix/ProtoChunkTickListFix.mapping

```diff
@@ -1,3 +1,7 @@
 CLASS net/minecraft/class_6740 net/minecraft/datafixer/fix/ProtoChunkTickListFix
+	FIELD field_35447 WATER_CONTAINING_BLOCK_IDS Lcom/google/common/collect/ImmutableSet;
```

Comment:
What is a water containing block? Is that different from a waterlogged block?

Reply:
The list contains kelp, seagrass, bubble columns, etc. Is there a word that describes these - blocks that are "always waterlogged"?

Reply:
Not sure, but I think `ALWAYS_WATERLOGGED_BLOCK_IDS` is more descriptive then.
```suggestion
	FIELD field_35447 ALWAYS_WATERLOGGED_BLOCK_IDS Lcom/google/common/collect/ImmutableSet;
```